datarepo-clienttests/src/main/java/common/commands/PrintHelp.java

```diff
@@ -115,7 +115,7 @@ public static void printHelp() throws IOException {
             + "Example Workflow (7): Execute a test configuration while locking the server, collect the measurements generated by the server during the run, and upload the results"
             + ANSI_RESET
             + System.lineSeparator()
-            + "export TEST_RUNNER_SERVER_SPECIFICATION_FILE=\"mmdev.json\"\n"
+            + "export TEST_RUNNER_SERVER_SPECIFICATION_FILE=\"localhost.json\"\n"
```

Comment:
why these changes? what is mmdev

Reply:
oh its a personal deployment? So this pr is not only removing references to integration but also to personal deployments, is that right?

Reply:
mmdev is an _old_ personal deployment for a Verily engineer